.github/workflows/_build_test_upload.yml

```diff
@@ -367,15 +366,6 @@ jobs:
             CONDA_CHANNEL=pytorch-${{ needs.get_release_type.outputs.type }}
           fi
           conda install pytorch torchdata cpuonly -c $(pwd)/conda-bld/ -c "$CONDA_CHANNEL"
-      - name: Run DataPipes Tests with pytest
-        shell: bash -l {0}
-        run: |
-          if ${{ ! startsWith( matrix.os, 'windows' ) }}; then
-            source "${MINICONDA_INSTALL_PATH}/etc/profile.d/conda.sh"
-          fi
-          conda activate "${CONDA_ENV_PATH}"
-          conda env update --file test/requirements.yml
-          python -m pytest --no-header -v test --ignore=test/test_period.py --ignore=test/test_text_examples.py --ignore=test/test_audio_examples.py
```

Comment:
Remove tests for conda because the smoketest has been carried out by `meta.yaml`

Reply:
This might have been mentioned before but when does `meta.yaml` gets executed? After binaries are built and before upload? And is it only for conda?

Reply:
It's a conda specific mechanism. 
`conda build` will execute the tests specified in `meta.yaml` after finishes building TorchData
 See: https://github.com/pytorch/data/actions/runs/3892182151/jobs/6643307007#step:8:2119